doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -167,48 +158,101 @@ Bluetooth samples
 
-    * Updated definitions for sensor callbacks to include the :c:struct:`bt_mesh_sensor_srv` context.
+  * :ref:`modem_shell_application` sample:
 
-Thread samples
---------------
+    * Added:
+
+      * An option ``--interval`` (in seconds) to neighbor cell measurements in continuous mode  (``link ncellmeas --continuous``).
+        When using this option, a new measurement is started in each interval.
+      * Separate plain AT command mode that can be started with the command ``at at_cmd_mode start``.
+        AT command termination methods can be configured using Kconfig options.
+        The default method is CR termination.
+        In AT command mode, a maximum of 10 AT commands can be pipelined with ``|`` as the delimiter character between pipelined AT commands.
+      * Threading support for the ``ping`` command.
+      * Iperf3 usage over Zephyr native TCP/IP stack and nRF9160 LTE default context.
+      * Support for the GNSS features introduced in modem firmware v1.3.2.
+        This includes several new fields in the PVT notification and a command to query the expiry times of assistance data.
+
+  * :ref:`nrf_cloud_rest_fota` sample:
+
+    * Updated:
+
+      * Enabled building of bootloader FOTA update files.
+      * Corrected handling of the bootloader FOTA updates.
+
+  * :ref:`lwm2m_client` sample:
+
+    * Added:
+
+      * Minimal Portfolio object support that is required for LwM2M conformance testing.
+      * Support for using :ref:`location_assistance` with Coiote LwM2M server.
+      * Guidelines on :ref:`setting up the sample for production <lwm2m_client_provisioning>` using AVSystem’s Coiote Device Management server.
+      * :ref:`state_diagram` to the sample documentation.
+
+    * Updated:
+
+      * Reworked the retry logic so that the sample can fall back to bootstrap mode and need not always restart the LTE connection.
+      * Replaced the deprecated GPS-driver with the new GNSS interface.
+
+  * :ref:`download_sample` sample:
+
+    * Updated the default HTTPS URL and certificate due to the old link being broken.
+
+OpenThread samples
+------------------
 
 * Updated:
 
   * Thread 1.2 version made the default configuration option.
+  * Thread Beacon payload has been removed after changes in latest Thread Specification.
   * The relevant sample documentation pages with information about support for :ref:`Trusted Firmware-M <ug_tfm>`.
   * All sample documentation with a Configuration section, and organized relevant information under that section.
-  * Thread Beacon payload has been removed after changes in latest Thread Specification.
 
 Matter samples
 --------------
 
 * Added:
 
   * Release configuration for all samples.
-  * A binding cluster to the Light Switch sample.
-  * The Groupcast communication to the Light Switch example.
-  * OTA DFU support for template application
+  * OTA DFU support for template application.
```

Comment:
This should go into a subsection dedicated to the template sample for Matter.
(:ref:`matter_template_sample` sample: Added OTA DFU support.)

Reply:
Fixed